src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+import core.basesyntax.model.OperationsList;
+import core.basesyntax.model.Transaction;
+import core.basesyntax.servise.DataOperationHandler;
+import core.basesyntax.servise.FileReader;
+import core.basesyntax.servise.impl.DataOperationHandlerImpl;
+import core.basesyntax.servise.impl.FileReaderImpl;
+import core.basesyntax.servise.impl.ReportGeneratorImpl;
+import core.basesyntax.servise.impl.ReportWriterImpl;
+import core.basesyntax.strategy.OperationStrategy;
+import java.io.File;
+import java.util.List;
+
+public class Main {
+    public static void main(String[] args) {
+        File input = new File("C:\\Users\\Thousandfold\\IdeaProjects"
+                + "\\jv-fruit-shop\\src\\main\\resources\\input");
```

Comment:
C:\\Users\\Thousandfold\\IdeaProjects"
                + "\\jv-fruit-shop\\src\\main\\resources\\input")

This will not work on other platforms
try to write: src/main/resources/input.csv